src/main/java/org/apache/commons/lang3/reflect/MemberUtils.java

```diff
@@ -260,12 +267,13 @@ private static boolean isMatchingExecutable(final Executable method, final Class
         }
 
         if (method.isVarArgs()) {
-            int i;
+            @NonNegative int i;
```

Comment:
Why do you need this annotation?

Reply:
If I don't annotate it to be `@NonNegative`, checker indicates array.access.unsafe.low error in line 279 if (!ClassUtils.isAssignable(parameterTypes[i], varArgParameterType, true)) {
                                                                                  ^

Reply:
Okay, this makes sense. Can you add a comment explaining that to this code - otherwise, seeing an annotation on a local variable is a surprise.